libs/datasets/sources/dh_beds.py

```diff
@@ -0,0 +1,145 @@
+import logging
+from collections import defaultdict
+import numpy
+import pandas as pd
+from libs.datasets.beds import BedsDataset
+from libs.datasets import dataset_utils
+from libs.datasets import data_source
+
+_logger = logging.getLogger(__name__)
+
+
+def match_county_to_fips(data, fips_data, county_key="county", state_key="state"):
+    county_combos = set(data.set_index([state_key, county_key]).index.to_list())
+    fips_combos = set(
+        fips_data.set_index([state_key, county_key, "fips"]).index.to_list()
+    )
+    fips_by_state_county = {
+        (
+            state,
+            county.lower()
+            .replace("'", "")
+            .replace("-", " ")
+            .replace("ó", "o")
+            .replace(".", "")
+            .replace("ñ", ""),
+        ): fips
+        for state, county, fips in fips_combos
+    }
+    counties_by_state = defaultdict(list)
+
+    for state, county, _ in fips_combos:
+        counties_by_state[state].append(
+            county.lower()
+            .replace("'", "")
+            .replace("-", " ")
+            .replace("ó", "o")
```

Comment:
would it be worth prefetching on some other accented chars? 
Also, is there a reason ó => o but ñ => ""? At least at glance it seems it should also go to a non-accented form?

Reply:
oh yeah, this was super hacky - I'll look into a better solution for this.  

https://stackoverflow.com/a/2633310

Huh this is cool (example from SO):
```python
accented_string = u'Málaga'
# accented_string is of type 'unicode'
import unidecode
unaccented_string = unidecode.unidecode(accented_string)
# unaccented_string contains 'Malaga'and is of type 'str'
```